code/src/Components/FirstQ.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+
+const FirstQuestion = ({name, setName,  nextSection, handleSubmit}) => {
+    return (
+        
+            <article >
+                <div className="text-group">
+                    <h1>Welcome to the music survey</h1>
+                    <label htmlFor="name" 
```

Comment:
Good job! 🥇 

If you would like, you can also add:
 id="name" in the <input-tag>, then you will automatically bump into the name-field when pressing the text "Enter your name"